docs/evm/guides/vrf.md

```diff
@@ -79,11 +85,13 @@ contract CadenceArchCaller {
 
 1. **Cadence Arch Address**:
 
-   The `cadenceArch` variable stores the address of the Cadence Arch precompiled contract (`0x0000000000000000000000010000000000000001`), which is constant across Flow EVM.
+   The `cadenceArch` variable stores the address of the Cadence Arch precompiled contract
+   (`0x0000000000000000000000010000000000000001`), which is constant across Flow EVM.
 
 2. **Revertible Random**:
 
-   The `revertibleRandom()` function makes a static call to the `revertibleRandom()` function to fetch a pseudo-random number. If the call is successful, it decodes the result as a `uint64` random value.
+   The `revertibleRandom()` function makes a static call to the `revertibleRandom()` function to fetch a pseudo-random
```

Comment:
[optional] not sure if we should precise the exact Cadence call, since the Cadence function is `revertibleRandom<type>() type`
```suggestion
   The `revertibleRandom()` function makes a static call to the `revertibleRandom<uint64>()` function to fetch a pseudo-random
```